### PR TITLE
feat: added tradetrust coo and invoice schema

### PR DESCRIPTION
**Pull request by ChenRen93**
- Added Certificate Of Origin and Invoice Schema for TradeTrust
- Added JSON-LD validation.

### CODE REVIEW

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts [hunk 1]
```diff
@@ -1,172 +1,76 @@
...
-          "message": "should have required property 'supplyChainConsignment'",
-          "params": Object {
-            "missingProperty": "supplyChainConsignment",
-          },
-          "schemaPath": "#/required",
-        },
-      ]
-    `);
+describe("certificate of origin schema", () => {
+  it("should work with valid certificate of origin", async () => {
```
Comment (Nebulis) [inline src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts:23]: this does not test the schema but the context

Comment (ChenRen93) [reply src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts:23]: okay, i will change it to context.test.ts

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts [hunk 2]
```diff
@@ -1,172 +1,76 @@
-import Ajv from "ajv";
-import schema from "./schema.json";
-import sampleDocument from "./certificate-of-origin-wrapped.json";
-import { cloneDeep, omit } from "lodash";
+import CertificateOfOriginSchema from "./CertificateOfOrigin.v3.json";
```
Comment (Nebulis) [inline src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts:1]: name is weird, it makes it sounds it's a schema but it's not

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts [hunk 3]
```diff
@@ -1,172 +1,76 @@
...
-          "message": "should have required property 'postalAddress'",
-          "params": Object {
-            "missingProperty": "postalAddress",
-          },
-          "schemaPath": "#/allOf/1/required",
-        },
-      ]
-    `);
+    return expandDocument(mergedDocument).catch(e => {
+      expect(e.message).toMatchInlineSnapshot(
```
Comment (Nebulis) [inline src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts:43]: https://jestjs.io/docs/tutorial-async#rejects

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts [hunk 4]
```diff
@@ -1,172 +1,76 @@
...
   });
 
-  it("should fail when document does not have importer's iD, name, postalAddress", () => {
-    const document = {
-      ...sampleDocument,
-      supplyChainConsignment: {
-        ...sampleDocument.supplyChainConsignment,
-        importer: {}
-      }
+  it("should throw error when unknown context or properties found", async () => {
```
Comment (Nebulis) [inline src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts]: what are we testing exactly? shouldn't be X or Y

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts (hunk 2 above)
Comment (ChenRen93) [reply src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts]: okay, changed to import context.

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts (hunk 1 above)
Comment (ChenRen93) [reply src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts]: done. updated description from schema to context

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts (hunk 3 above)
Comment (ChenRen93) [reply src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts]: done. updated test to read .rejects and expect throwError

File: src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts (hunk 4 above)
Comment (ChenRen93) [reply src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts]: done.

File: src/io/tradetrust/invoice/1.0/invoice-data.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
+    "billFrom": {
```
Comment (Nebulis) [inline src/io/tradetrust/invoice/1.0/invoice-data.json:2]: eslint doesnt work on json ? can check ? (should be 2 spaces, not four)

**Pull request merged**